index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Mobile site</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans|Roboto+Mono&display=swap" rel="stylesheet">
+</head>
+
+<body>
+<header>
+    <div class="intro">
+        <h1 class="motto">Reliable, efficient delivery</h1>
+        <h1>Powered by Technology</h1>
+        <p class="quote">Our Artificial Intelligence powered tools
+            use millions of project data points
+            to ensure that your project is successful</p>
+    </div>
+</header>
+
+<div class="box-container">
+    <div class="box-1">
+        <h2>Supervisor</h2>
+        <p>Monitors activity to identify project roadblocks</p>
+        <img class="images" src="images/icon-supervisor.svg" alt="magnifying glass with eye" />
```

Comment:
cool, you so use like that, I didn't know it was possible, I put the complete  link on my code, now it is enormous.

Reply:
I got that from that tutorial I mentioned in the previous comment. I created a new directory (folder) within the parent folder, labelled it 'images', downloaded the images onto comp then moved them to that images folder I created :-)